Dockerfile.dev

```diff
@@ -58,5 +58,7 @@ RUN apt-get install chromium --fix-missing -y
 COPY . .
 RUN cd public/creators-landing && yarn install && yarn build
 
+RUN NODE_OPTIONS=--openssl-legacy-provider RAILS_ENV=development CREATORS_FULL_HOST="1" SECRET_KEY_BASE="1" bundle exec rails assets:precompile 
```

Comment:
I think we'll want to leave the env production here for compiling, even in the dev docker since the prod env is set up to correctly compile and that's the only ENV we actually compile in.

Reply:
So then this just becomes a sanity check when developing.